### DISCUSSION TITLE
I have 2PIB plots，but it hasn't exploded in 4 days，what is the reason

### DISCUSSION THREAD

**Discussion by KmBooy** [Chia-Network/chia-blockchain#5970] [Support]
I have 3 piB of memory power, but I haven't exploded in 4 days.What is the reason?My wallet has been connected to the main network and it shows 50+ nodes。
![image](https://user-images.githubusercontent.com/35790216/119435330-7e5b2a00-bd4c-11eb-9c83-084f17d2b655.png)
There are about 30 harvesters
![image](https://user-images.githubusercontent.com/35790216/119435418-a9de1480-bd4c-11eb-919c-fb4f62f70bc0.png)
![image](https://user-images.githubusercontent.com/35790216/119435555-ef024680-bd4c-11eb-9076-2d2ab98f55b2.png)

there are 50 nodes

Computer version: Windows 10
Chia version: 1.1.6

**Comment by KmBooy** [comment]
![image](https://user-images.githubusercontent.com/35790216/119435811-86679980-bd4d-11eb-8c4c-ea7223c5e166.png)
Previous record

**Comment by losingle** [comment]
现在大部分都碰到这个问题了
你这个应该是收割机运行不正常,你把收割机的日志贴出来

**Reply by coinsccg** [reply]
> 现在大部分都碰到这个问题了
> 你这个应该是收割机运行不正常，你把收割机的日志贴出来
The harvester is normal, 2.5PB doesn't produce XCH in three days, which is very abnormal. Is it because of version 1.1.6?

**Reply by KmBooy** [reply]
Where to Find the harverster log

**Comment by Hugh-Top** [comment]
我有2pb，也有三天没出块了，日志也没看出什么异常

**Reply by coinsccg** [reply]
> 我有2pb，也有三天没出块了，日志也没看出什么异常

我有2.5Pb, 三天没出块了，日志也是正常的，你更新到1.1.6了吗 ，我的版本是1.1.6

**Reply by Hugh-Top** [reply]
我也是最新版本1.1.6

**Reply by coinsccg** [reply]
> 我也是最新版本1.1.6

就很奇怪，我看到大部分最近两三天不出币的都介于2-3p之间，那些1-2P的人，还是出了的

**Reply by Hugh-Top** [reply]
我还一个节点，2.x P，24小时内出了2个块，这个很奇怪

**Reply by coinsccg** [reply]
难道这就是拼运气？p图量是摆设吗

**Reply by KmBooy** [reply]
We need to find the authorities.

**Reply by coinsccg** [reply]
> 我们需要找到当局。

你现在出了吗

**Reply by Hugh-Top** [reply]
how to find authorities???

**Reply by KmBooy** [reply]
I still haven't. Today is the fourth day

**Reply by losingle** [reply]
被攻击了

**Reply by coinsccg** [reply]
> 被攻击了
确定吗，有攻击日志吗

**Reply by ben00007** [reply]
> 被攻击了
怎么确认是否被攻击了？

**Reply by ben00007** [reply]
I have 30PIB plots，but it hasn't exploded in 12hour，the version  is 1.1.6

**Reply by Hugh-Top** [reply]
> I have 30PIB plots，but it hasn't exploded in 12hour，the version is 1.1.6
你出块了吗

**Comment by ben00007** [comment]
I have 30PIB plots，but it hasn't exploded in 12hour，the version is 1.1.6

**Comment by KmBooy** [comment]
所有收割机重新更新了一遍，今天中币了，日志出现了一下问题
![image](https://user-images.githubusercontent.com/35790216/119587564-875a0300-be01-11eb-97da-1150fd323210.png)

**Reply by Hugh-Top** [reply]
这个日志只是表示，你计算quality的时间有点长了，并不能说明不出快的原因

**Comment by BrandtH22** [comment]
@KmBooy , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
